piglow/examples/strobe.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})
```

Comment:
It would be best to have the address exported from the package. There are 2 reasons for that: it's easier for the user but also the API is about to change and  the address is being removed from the Devfs type.